src/validators/fetch-todo-validator.ts

```diff
@@ -0,0 +1,8 @@
+import { param, ValidationChain } from 'express-validator'
+import { AppContext } from '@typings'
+
+const fetchTodoValidator = (appContext: AppContext): ValidationChain[] => [
+  param('id', 'VALIDATION_ERRORS.INVALID_ID').isMongoId(),
+]
+
+export default fetchTodoValidator
```

Comment:
Please do make sure you are leaving a new line by EOF for all files which are created (You can use EditorConfig - https://editorconfig.org/ which would do this automatically for you)